### PR TITLE
Add current assets to small full ixbrl template

### PR DESCRIPTION
**Pull request by leahtaylor**
Add current assets to ixbrl model
Add current assets and logic to template

Current Assets
- Stocks
    - current amount
    - previous amount
- Debtors
    - current amount
    - previous amount
- Cash at bank and in hand
    - current amount
    - previous amount
current current assets total
previous current assets total

SFA-710, SFA-461

### CODE REVIEW

**Comment by leahtaylor** [comment]
screen shot 2018-10-16 at 12 12 46

screen shot 2018-10-16 at 11 55 50

File: src/main/resources/filing/templates/small-full-accounts.html [hunk 1]
```diff
@@ -1733,6 +1734,103 @@ <h2 class="text--center">Balance sheet</h2>
...
+
+                <tr>
+                  <th class = "strong" id="current-assets-heading">Current assets</th>
+                </tr>
+
+                <!-- BALANCE SHEET - CURRENT ASSETS - STOCKS -->
+
+              {{if $stocks := $currentAssets.stocks}}
+                <tr class="figures">
+                  <td id="stocks-label">Stocks:</td>
```
Comment (ch-mcabrera) [inline src/main/resources/filing/templates/small-full-accounts.html]: Do we need to match our **id**'s to the **id** in the SmallFullAccountsModifiediXBRL.html?

Ours: `<td id="stocks-label">Stocks:</td>`
SmallFullAccountsModifiediXBRL: `<td id="ac50-label">Stocks:</td>`

File: src/main/resources/filing/templates/small-full-accounts.html [hunk 2]
```diff
@@ -1733,6 +1734,103 @@ <h2 class="text--center">Balance sheet</h2>
...
+
+                  <td class="figure noteIndex" id="uk-busTotalInventoriesCY_ENDNoteIndexes"></td>
+
+                  <td id="stocks-curr-val" class="strong figure">
```
Comment (ch-mcabrera) [inline src/main/resources/filing/templates/small-full-accounts.html]: Id's mistmatched
Ours: <td id="stocks-curr-val" class="strong figure">
SmallFullAccountsModifiediXBRL: <td id="ac50-curr-val" class="strong figure">

File: src/main/resources/filing/templates/small-full-accounts.html [hunk 3]
```diff
@@ -1733,6 +1734,103 @@ <h2 class="text--center">Balance sheet</h2>
...
+                    {{if isNotEmpty $stocks.current_amount}}
+                      <ix:nonFraction contextRef="CY_END" decimals="0" unitRef="GBP" name="core:TotalInventories" format="ixt2:numdotdecimal">{{$stocks.current_amount}}</ix:nonFraction>
+                    {{end}}
+                  </td>
+
+                  <td id="stocks-prev-val" class="figure">
```
Comment (ch-mcabrera) [inline src/main/resources/filing/templates/small-full-accounts.html]: Same here, Id's mistmatched:
Ours: `<td id="stocks-prev-val" class="figure">`
SmallFullAccountsModifiediXBRL: `<td id="ac50-prev-val" class="figure">`

File: src/main/resources/filing/templates/small-full-accounts.html [hunk 4]
```diff
@@ -1733,6 +1734,103 @@ <h2 class="text--center">Balance sheet</h2>
...
+                  <th class = "strong" id="current-assets-heading">Current assets</th>
+                </tr>
+
+                <!-- BALANCE SHEET - CURRENT ASSETS - STOCKS -->
+
+              {{if $stocks := $currentAssets.stocks}}
+                <tr class="figures">
+                  <td id="stocks-label">Stocks:</td>
+
+                  <td class="figure noteIndex" id="uk-busTotalInventoriesCY_ENDNoteIndexes"></td>
```
Comment (ch-mcabrera) [inline src/main/resources/filing/templates/small-full-accounts.html]: Do we know when we are adding the ordering number? they are in the template; however, this has not been added to any of the notes yet.

Comment (leahtaylor) [reply src/main/resources/filing/templates/small-full-accounts.html]: These will be when we start adding the corresponding balance sheet notes

File: src/main/resources/filing/templates/small-full-accounts.html (hunk 3 above)
Comment (leahtaylor) [reply src/main/resources/filing/templates/small-full-accounts.html]: Carl palfrey has confirmed that Ids do not need to match the prototype- along as they are unique

**Pull request merged**